redbeams/src/main/java/com/sequenceiq/redbeams/controller/v4/database/DatabaseV4Controller.java

```diff
@@ -32,10 +33,13 @@
     @Inject
     private DatabaseConfigService databaseConfigService;
 
+    @Inject
+    private DatabaseConnectionValidator connectionValidator;
```

Comment:
This field is unused.

Reply:
Thanks. There once was a test to check for unused fields. I check with others if that still works.